config/sample.toml

```diff
@@ -139,4 +140,4 @@ netapp_qtree_name = "bai_qtree" # default qtree name for backend.ai exclusive
 netapp_xcp_container_name = "netapp-xcp" # only required when xcp activated by container
 netapp_xcp_hostname = "xcp-hostname"
 # default xcp catalog path goes to the directory named "catalog" of the first NetApp volume
-netapp_xcp_catalog_path = "path for xcp-catalog" # Hint: execute command cat /opt/NetApp/xFiles/xcp/xcp.ini and see nfs mount path
\ No newline at end of file
+netapp_xcp_catalog_path = "path for xcp-catalog" # Hint: execute command cat /opt/NetApp/xFiles/xcp/xcp.ini and see nfs mount path
```

Comment:
@fregataa I can't see any significant difference in this statement. Could you explain what's been changed?

Reply:
I added a new line. nothing changed except that